blockstore.go

```diff
@@ -28,58 +29,17 @@ var ErrHashMismatch = errors.New("block in storage has different hash than reque
 // ErrNotFound is an error returned when a block is not found.
 var ErrNotFound = errors.New("blockstore: block not found")
 
-// Blockstore wraps a Datastore block-centered methods and provides a layer
-// of abstraction which allows to add different caching strategies.
-type Blockstore interface {
-	DeleteBlock(cid.Cid) error
-	Has(cid.Cid) (bool, error)
-	Get(cid.Cid) (blocks.Block, error)
+// Blockstore aliases upstream blockstore interface
+type Blockstore ib.Blockstore
 
-	// GetSize returns the CIDs mapped BlockSize
-	GetSize(cid.Cid) (int, error)
+// GCLocker aliases upstream gclocker interface
+type GCLocker ib.GCLocker
 
-	// Put puts a given block to the underlying datastore
-	Put(blocks.Block) error
+// GCBlockstore aliases upstream gcblockstore interface
+type GCBlockstore ib.GCBlockstore
```

Comment:
use = as in `type TypeName = ib.TypeName` for type alias.

Reply:
good catch